templates/partials/contents.html

```diff
@@ -21,6 +22,18 @@ <h2>{{item.title}}</h2>
     <p>{{item.description}}</p>
   {% endif %}
 
+
+  {%- if item.guidance -%}
+    {% set contents = item.guidance.contents %}
+      {% call onsPanel({
+      "id": "item-guidance-" ~ guidance_id,
+      "classes": "ons-u-mb-m"
+      }) %}
+        {% set contents = item.guidance.contents %}
+        {% include 'partials/contents.html' %}
+      {% endcall %}
+  {% endif %}
```

Comment:
As per my comment on validator, we should handle this in `basic.html`. Introduction primary content is already an array so it gives them the flexibility they need which allowing guidance (panel) in generic content blocks.

Reply:
Context for others. Moving back to contents, since we already allow definitions inside contents, which in itself is recursive. If it becomes a problen, we will revisit.